DEVELOPMENT.md

```diff
@@ -70,3 +70,39 @@ or directly.
 A `.clang-format` file which defines our specific code style lives in the
 top level directory of the repository. The style is identical to the Google
 style.
+
+## Cross-Compiling for GGP
+
+_Note:_ This was only tested on Linux. But cross compilation on Windows
+should work as well.
```

Comment:
Shouldn't this be tested on Windows as well if it's also supposed to work there?

Reply:
Good point. I'm not sure if it is "supposed" to work there. But maybe it's a good idea. I will look into it. I just hope it won't be another rabbit hole. ;-)

Reply:
I looked into this.... Aaaannd, it is another rabbit hole. So for now, no Windows support.